std/algorithm/sorting.d

```diff
@@ -2200,6 +2203,13 @@ auto topN(alias less = "a < b",
             swap(r.front, r.minPos!less.front);
             break;
         }
+        if (nth + 1 == r.length)
+        {
+            // Special-case "max"
+            import std.algorithm.searching : minPos;
```

Comment:
Shouldn't this be merged with the same import in the previous if-block and hoisted one lexical scope up?